cfgov/cfgov/settings/production.py

```diff
@@ -114,3 +115,10 @@
         'TIMEOUT': None,
     }
 }
+
+# ALLOWED_HOSTS should be defined as a JSON list in the ALLOWED_HOSTS
+# environment variable.
+try:
+	ALLOWED_HOSTS = json.loads(os.getenv('ALLOWED_HOSTS'))
+except ValueError:
```

Comment:
If the environment variable isn't set, then `json.loads(None)` raises a `TypeError` instead of a `ValueError`. How would you feel about catching both types of exceptions (`except (TypeError, ValueError):`)?